### PR TITLE
11638_create_admin_page

### PR DESCRIPTION
**Pull request by phongtt-1837**
https://edu-redmine.sun-asterisk.vn/issues/11638

### CODE REVIEW

**Comment by phongtt-1837** [comment]
ready

File: resources/views/admin/navbar.blade.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                            </li>
+                            <li><a class="menu-item" href="{{ route('courses.create') }}" data-i18n="nav.templates.horz.top_icon">{{ __('Create New Course') }}</a>
+                            </li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+    </div>
+</div>
```
Comment (BunnyPi04) [inline resources/views/admin/navbar.blade.php]: thiếu dòng trắng cuối file

File: routes/web.php [hunk 2]
```diff
@@ -17,3 +17,13 @@
 Auth::routes(['register' => false]);
 
 Route::get('/home', 'HomeController@index')->name('home');
+
+Route::group(array('prefix' => 'admin', 'namespace' => 'Admin','middleware' => 'admin'), function () {
```
Comment (BunnyPi04) [inline routes/web.php]: convention

**Comment by phongtt-1837** [comment]
ready

**Pull request merged**